src/main/java/com/github/tomakehurst/wiremock/common/xml/Xml.java

```diff
@@ -116,7 +116,7 @@ private static TransformerFactory createTransformerFactory() {
                   .getDeclaredConstructor()
                   .newInstance();
```

Comment:
On JDK17 this throws `java.lang.IllegalAccessException: class com.github.tomakehurst.wiremock.common.xml.Xml cannot access class com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl (in module java.xml) because module java.xml does not export com.sun.org.apache.xalan.internal.xsltc.trax to unnamed module @96532d6`.

Can't it be removed? Similar to how it's done here:

https://github.com/wiremock/wiremock/blob/fbf904da993830e6f16f4fb8d6d8e694d6645dae/src/main/java/com/github/tomakehurst/wiremock/extension/responsetemplating/helpers/FormatXmlHelper.java#L68-L69